### ISSUE TITLE
Jailer of Justice doesn't spawn

### ISSUE THREAD

**Issue by przemoc1** [DarkstarProject/darkstar#5756]
**_I have:_**

- [x] searched existing issues (http://github.com/darkstarproject/darkstar/issues/) to see if the issue I am posting has already been addressed or opened by another contributor
- [x] checked the commit log to see if my issue has been resolved since my server was last updated

**_Client Version_** (type `/ver` in game) **:30190202_0** 

**_Source Branch_** (master/stable) **:master** 

**_Additional Information_** (Steps to reproduce/Expected behavior) **:Trading required items to ??? at the place brings no effect (??? is not accepting traded items).**

**Comment by ibm2431** [comment]
[Jailer of Justice spawning seems to be commented out](https://github.com/DarkstarProject/darkstar/blob/master/scripts/zones/AlTaieu/npcs/qm2.lua). Its [mob file looks mostly empty](https://github.com/DarkstarProject/darkstar/blob/master/scripts/zones/AlTaieu/mobs/Jailer_of_Justice.lua); I'd wager it is not fully implemented yet.

**Comment by TeoTwawki** [comment]
its default disabled for loot pinata reasons. so are several other NM in SEA

**Comment by KnowOne134** [comment]
Can close as the mob is just not coded to standards

**Comment by TeoTwawki** [comment]
I left it open so it doewsn't get re-reported

**Comment by KnowOne134** [comment]
Fair enough. I have it updated on my server. Maybe I'll just pr it and we can close. Unless you want to do them all 1st

**Comment by TeoTwawki** [comment]
Bring justice down upon him, let no loot pinata escape.

PR for great justice, launch every zig.
